src/trumbowyg.js

```diff
@@ -805,7 +805,7 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                         $dropdown.append(t.buildSubBtn(def));
                     }
                 });
-                t.$box.append($dropdown.hide());
+                $btn = $btn.add($dropdown.hide());
```

Comment:
Why not just `$btn.append` here?

Reply:
My dated javascript knowledge...  :/
I can do a further commit to use append or you can double check nothing is broken under this concpot first?